core/tests/protractor_utils/waitFor.js

```diff
@@ -18,7 +18,7 @@
  */
 
 var until = protractor.ExpectedConditions;
-var DEFAULT_WAIT_TIME_MSECS = 10000;
+var DEFAULT_WAIT_TIME_MSECS = browser.isMobile ? 20000 : 10000;
 
```

Comment:
20 seconds? Are things that bad on mobile? I assume things are going haywire with just 10 secs? 
( No action needed here, just curious )

Reply:
The tests run on a real remote mobile device. This isn't about being bad, it's just that the connection might require some time. Therefore, I have added this as a precaution.

Reply:
Got it!